src/main/EncryptorDecryptor.java

```diff
@@ -0,0 +1,96 @@
+package main;
+
+import main.algorithms.Algorithm;
+import main.algorithms.ShiftAlgorithmImpl;
+import main.algorithms.UnicodeAlgorithmImpl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class EncryptorDecryptor {
+    private String mode;
+    private String data;
+    private int key;
+    private String in;
+    private String out;
+    private Algorithm algorithm;
+
+    public EncryptorDecryptor() {
+        mode = "enc";
+        data = "";
+        key = 0;
+        out = "";
+    }
+
+
+    public void invoke() {
+        readFromFile();
+        String result = switch (this.mode) {
+            case "enc" -> algorithm.encrypt(data, key);
+            case "dec" -> algorithm.decrypt(data, key);
+            default -> "";
+        };
+
+        writeToFile(result);
+    }
+
```

Comment:
The same here. Please remove the redundant row.

Reply:
Fixed.